db/repos/blocks.js

```diff
@@ -23,110 +25,238 @@ var columnSet;
+	 * @param {int} params.height
+	 * @param {int} params.limit
+	 * @return {Promise}
+	 */
+	getIdSequence (params) {
+		// TODO: Must use a result-specific method, not .query
+		return this.db.query(sql.getIdSequence, params);
+	}
+
+	/**
+	 * Get common block among peers
+	 * @param {Object} params
+	 * @param {string} params.id
+	 * @param {string} params.previousBlock
+	 * @param {int} params.height
+	 * @return {Promise}
+	 */
+	getCommonBlock (params) {
+		// TODO: Must use a result-specific method, not .query
+		params.comparePreviousBlock = params.previousBlock ? this.pgp.as.format('AND "previousBlock" = ${previousBlock}', params): '';
+		return this.db.query(sql.getCommonBlock, params);
+	}
+
+	// TODO: Merge BlocksRepository#getHeightByLastId with BlocksRepository#list
+	/**
+	 * Get height of the block with id
+	 * @param {string} lastId - Id of the block to search
+	 * @return {Promise}
+	 */
+	getHeightByLastId (lastId) {
+		// TODO: Must use a result-specific method, not .query
+		return this.db.query(sql.getHeightByLastId, [lastId]);
+	}
+
+	/**
+	 * Load block including transactions
+	 * @param {Object} params
+	 * @param {string} params.id
+	 * @param {string} params.lastId
+	 * @param {int} params.height
+	 * @param {int} params.limit
+	 * @return {Promise}
+	 */
+	loadBlocksData (params) {
+		// TODO: Must use a result-specific method, not .query
+		return this.db.query(Queries.loadBlocksData(params), params);
+	}
+
+	/**
+	 * Load blocks including transactions with an offset and limit
+	 * @param {int} offset
+	 * @param {int} limit
+	 * @return {Promise}
+	 */
+	loadBlocksOffset (offset, limit) {
+		// TODO: Must use a result-specific method, not .query
+		return this.db.query(sql.loadBlocksOffset, [offset, limit]);
+	}
+
+	/**
+	 * Load the last block including transactions
+	 * @return {Promise}
+	 */
+	loadLastBlock () {
+		// TODO: Must use a result-specific method, not .query
+		return this.db.query(sql.loadLastBlock);
+	}
+
+	/**
+	 * Check if a block exits with a particular ID
+	 * @param {string} id
+	 * @return {Promise}
+	 * @throws {QueryResultError} - Multiple rows were not expected - in the case of multiple blocks found with same id
+	 */
+	blockExists (id) {
+		return this.db.oneOrNone(sql.blockExists, [id]);
+	}
+
+	/**
+	 * Delete all blocks after a particular height
+	 * @param {string} id
+	 * @return {Promise}
+	 */
+	deleteAfterBlock (id) {
+		return this.db.none(sql.deleteAfterBlock, [id]);
+	}
+
+	/**
+	 * Get multiple blocks to be transported to peers
+	 * @param {string} ids - Comma separated string of ids
+	 * @return {Promise}
+	 */
+	getBlocksForTransport (ids) {
+		// TODO: Must use a result-specific method, not .query
+		return this.db.query(sql.getBlocksForTransport, [ids]);
+	}
+
+	/**
+	 * Insert a block to database
+	 * @param {Object} block - Attributes to be inserted, can be any of [BlocksRepo's dbFields property]{@link BlocksRepo#dbFields}
```

Comment:
Notice, you removed this property `dbFields` and forgot to updated the reference. Also name of the repo is mismatched.